ReadAlongs/Studio-Web
# fix: bump nx to 18.3.4 so it works on windows again

Fixes #289



### PR Goal? 

Get `npx nx cmd package` working on Windows again

### Fixes? 

Fixed #289

### Feedback sought? 

If you have a Windows machine and had the problem before, test that this fixes it.

### Priority? 

normal

### Tests added? 

n/a

### How to test? 

Run `npx nx run-many --targets=serve-test-data,serve --projects=web-component` in one shell
Run `npx nx test:open web-component` in another.

On Windows before this PR, the second command would hang and be impossible to abort except by killing the node process.
With this PR, testing runs, and Ctrl-C aborts as expected.

### Confidence? 

high

### Version change? 

nope
